parity-crypto/Cargo.toml

```diff
@@ -15,7 +15,7 @@ required-features = ["publickey"]
 
 [dependencies]
 tiny-keccak = "1.4"
-eth-secp256k1 = { git = "https://github.com/paritytech/rust-secp256k1", rev = "a96ad75", optional = true }
+parity-secp256k1 = { version = "0.6.0", optional = true }
```

Comment:
This will likely be `0.7` when https://github.com/paritytech/rust-secp256k1/pull/29 lands.

Reply:
Ok, let me know, when you publish the new version, so I can proceed with this one.

Reply:
Changed